### ISSUE TITLE
Identity creation fail

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#4237]
### What happened?

Not possible to ceate identity

### Please reproduce in steps

Getting error after filling required info

### Expected Behavior

Creating identity without errors 😄

### What browsers are you seeing the problem on?

Chrome

### At which address did you encounter bug?

kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

PolkadotJS

### At which chain did you encounter bug?

RMRK

### Screenshots

![EB7CD5E6-09E6-4E77-91F0-ACC6850AA6D3](https://user-images.githubusercontent.com/104380776/199534241-f0259c74-eedc-4ed1-a739-ae94988287d3.jpeg)

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by yangwao** [comment]
can someone check if identity works ?

**Comment by JustLuuuu** [comment]
> can someone check if identity works?


it is working now, I just tried it. 

Btw I filled in my identity completely: my handle, name, web, Twitter, and email. But I only see the Twitter link when I hover over my name. 
<img width="324" alt="image" src="https://user-images.githubusercontent.com/104380776/213159783-5f10d185-8ffe-4906-995a-34040be468e2.png">

Also, I cannot see these things in my profile. It would be nice to have them there (from the perspective of artists).
